pkg/system/phase2_creating.go

```diff
@@ -574,6 +584,20 @@ func (r *Reconciler) SetDesiredCoreApp() error {
 		}}
 		util.MergeVolumeList(&podSpec.Volumes, &secretVolumes)
 	}
+
+	// We are binding bucket logging pvc to the core pod here
+	// TODO: need to update the logic and actual params here, currently hardcoded
+	if false {
+		bucketLogVolumes := []corev1.Volume{{
+			Name: "aayush-pvc",
+			VolumeSource: corev1.VolumeSource{
+				PersistentVolumeClaim: &corev1.PersistentVolumeClaimVolumeSource{
+					ClaimName: "aayush-pvc",
```

Comment:
I would move this name to the reconciler as there we will set the pvc name:
So add something like BucketLoggingPvc to there, and use it here

Reply:
Sure.